src/Compilers/CSharp/Portable/Syntax/SyntaxNodeExtensions.cs

```diff
@@ -109,5 +109,29 @@ internal static SyntaxToken ExtractAnonymousTypeMemberName(this ExpressionSyntax
                 }
             }
         }
+
+        internal static TypeSyntax SkipRef(this TypeSyntax syntax, out RefKind refKind)
+        {
+            refKind = RefKind.None;
```

Comment:
Actually, no it cannot. If I move it down then it will overwrite the assignment in the if block.

---
In reply to: [73443834](https://github.com/dotnet/roslyn/pull/12841#discussion_r73443834) [](ancestors = 73443834,73057131)